kuma/static/styles/components/newsletter.scss

```diff
@@ -61,13 +61,13 @@ input.newsletter-input-email {
     position: relative;
     width: 100%;
     padding: ($grid-spacing / 2);
-    @include bidi(((padding-right, 2em, padding-left, inherit),));
-    text-align: left;
+    @include bidi(((text-align, left, right),));
 
     .icon-arrow {
         position: absolute;
         top: 30%;
-        right: 10px;
+        @include bidi(((right, 10px, left, 10px),));
+        @include bidi(((left, auto, right, 10px),));
```

Comment:
IIUC, in ltr, this becomes:

```
right: 10px;
left: auto;
```

And in RTL, it becomes:

```
left: 10px;
right: 10px;
right: auto:
left: 10px;
```

So it seems like the 3-argument version of the bidi macro might make more sense here? Or am I misunderstanding something?

Also, given the nature of the output produced by bidi() is it more efficient to call bidi() once and pass both lists to a single invocation? (I don't know if it makes a difference, but given that it is an option, I'm wondering about it.)